lib/rubocop/node_pattern.rb

```diff
@@ -530,7 +530,7 @@ def node_search(method_name, compiler, on_match, prelude, called_from)
         src = node_search_body(method_name, compiler.emit_trailing_params,
                                prelude, compiler.match_code, on_match)
         filename, lineno = *called_from
-        class_eval(src, filename, lineno.to_i)
+        class_eval(src, filename, Integer(lineno))
```

Comment:
Builds on WIndows are failed, and the cause looks this line.

Reply:
Yep. `#to_i` -> `Integer` is not a safe auto-correct.